### ISSUE TITLE
Font Size of Primary menu to fit the Colour contrast requirement

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#76]
The Primary menu is only 12pts now. The visited link will fail the colour contrast requirement if it is too small. Making it 14 pt Bold will resolve this issue.

This is the px to point to em converter for reference : http://pxtoem.com/

![image](https://cloud.githubusercontent.com/assets/15111528/10954299/b1f05f98-83a0-11e5-98cd-54a75fa0dee4.png)

**Comment by HerinHentry** [comment]
@christosptr : Do you agree to this?

**Comment by christosptr** [comment]
yes I agree

**Comment by HerinHentry** [comment]
Alternative Colour suggestions 
Background : #393939
Links : #83CDBC
Active links : #F88A52 / #FF884C

But, this will fail with the Gradient. :(
So we might need another on focus / hover colour as well.

![image](https://cloud.githubusercontent.com/assets/15111528/10998970/4ee83904-84ed-11e5-9bb4-b27f7ad627d5.png)
![image](https://cloud.githubusercontent.com/assets/15111528/10999089/35cbe726-84ee-11e5-9761-e65fa47d75b5.png)

**Comment by mr-bobz** [comment]
Pls check again, changed whole menu theme to avoid violations!

**Comment by mr-bobz** [comment]
14pt would be too large for the menu!

**Comment by christosptr** [comment]
the new menu is fine now
I see not high contrast violations

Cheers Christos
Send from my mobile device

**Comment by HerinHentry** [comment]
There is no contrast violations for the foreground colours used & background. We are meeting the contrast 3:1.
But, @christosptr  Can you please advise on this? 
**G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them**.
When we say surrounding text, is this including other meu items?

**Comment by christosptr** [comment]
verified
